### PR TITLE
Add "dungeon_master" custom feature tag. Show/Hide host button depend…

### PR DESCRIPTION
**Pull request by richardwolny**
…ing on flag value.

### CODE REVIEW

File: root.tscn [hunk 1]
```diff
@@ -278,10 +278,10 @@ __meta__ = {
...
 
 [node name="color" type="ColorPicker" parent="menu/center/panel"]
-margin_left = 4.0
-margin_top = 4.0
-margin_right = 4.0
-margin_bottom = 4.0
+margin_left = 8.0
+margin_top = 8.0
+margin_right = 8.0
+margin_bottom = 8.0
```
Comment (AsherGlick) [inline root.tscn:284]: Is this change intentional?

Comment (richardwolny) [reply root.tscn:284]: Not really. These values kept changing whenever I opened Godot.
